src/flownet/templates/RSVD.jinja2

```diff
@@ -1,7 +1,12 @@
 RSVD
-{% for i in range(nr_eqlnum) -%}
+{% for i in range(1, nr_eqlnum + 1) -%}
+{%- if nr_rsvd > 1 -%}
+{%- for _, row in rsvd[rsvd['eqlnum']==i].iterrows() -%}
+{{ row['depth'] }} {{ row['rs'] }}
+{% endfor %}/
+{%- else -%}
 {%- for _, row in rsvd.iterrows() -%}
 {{ row['depth'] }} {{ row['rs'] }}
 {% endfor %}/
+{% endif %}
```

Comment:
there is one blank line appearing between the tables of the multiple regions when using a single RSVD table is provided in the `rsvd.csv` file. It due to one of the lines above in the jinja file

Reply:
Re-wrote the code a bit so that there is no if-statement in the Jinja template file. Should be better now.